chemie/customprofile/models.py

```diff
@@ -100,6 +100,7 @@ class Profile(models.Model):
     approved_terms = models.BooleanField(default=False)
 
     voted = models.BooleanField(default=False)
+    eligible_for_voting = models.BooleanField(default=False)
```

Comment:
Litt seint no, men det er vanlig å "decouple" apps ved å heller lage modeller i den spesifikke appen, enn å endre på modeller i andre apps. 
Litt som at vi ikke endrer på User (se linje 77), men har en OneToOne relation til User fra Profile.

Reply:
Ja. Det ble gjort for å ikke endre veldig mye i election, og luftet det på slack, men det ble nedstemt.